myapp/spec/models/tasks_spec.rb

```diff
@@ -5,50 +5,51 @@
 RSpec.describe "Tasks", type: :model do
   describe "validation" do
     context "create a task" do
+      let!(:user1) { User.create(name: "user 1", email: "user1@example.com", password: "123") }
       context "maximum length" do
-        let!(:task) { Task.create(title: "X" * 100, description: "Y" * 30000) }
+        let!(:task) { Task.create(title: "X" * 100, description: "Y" * 30000, user_id: user1[:id]) }
```

Comment:
```suggestion
        let!(:task) { Task.create(title: "X" * 100, description: "Y" * 30000, user_id: user1.id) }
```
属性にアクセスするときはこう書くほうが Ruby っぽいです

Reply:
こちらもXXX.YYYという形に全て変更しました